packages/cln_plugin/lib/src/rpcmethods/iclnrpcmethod.dart

```diff
@@ -0,0 +1,9 @@
+abstract class ICLNRpcMethod {
+  String getName();
+
+  String getUsage();
+
+  String getDescription();
+
+  String getLongDescription();
+}
```

Comment:
What is the reason for this abstract class? maybe can be removed?

The interface is good when we want add dependency injection to make the package testable, but here we can avoid this complexity that I added in the Java package!

Sorry for that!